src/pdf.js

```diff
@@ -0,0 +1,36 @@
+const pdf = require('pdf-parse');
+
+async function getPdfText(commentBody) {
+    const pdfUrl = getPdfUrl(commentBody);
+
+    let res = await fetch(pdfUrl);
+    const fileBuffer = await res.arrayBuffer();
+    
+    const pdfData = await pdf(fileBuffer, { pagerender: renderPage });
+    
+    return pdfData.text;
+}
+
+function getPdfUrl(commentBody) {
+    const match = commentBody.match(/\[pd\.pdf\]\(([^)]+)\)/);
```

Comment:
This grabs anything between `[pd.pdf](` and `)` so that we get the URL but based on your suggestion of using regex to look for `.pdf` instead in the comment, I'm going to now change this to grabbing between `.pdf](` and `)`.

Reply:
I see. Worth trying to understand the regex even if it's AI generated :)

I think it's generally fine, I'd only change the pd into .* to allow any PDF names

Reply:
Done.